itest/lnd_sweep_test.go

```diff
@@ -2154,3 +2154,151 @@ func runBumpFee(ht *lntest.HarnessTest, alice *node.HarnessNode) {
 	// Clean up the mempol.
 	ht.MineBlocksAndAssertNumTxes(1, 2)
 }
+
+// testBumpForceCloseFee tests that when a force close transaction, in
+// particular a commitment which has no HTLCs at stake, can be bumped via the
+// rpc endpoint `BumpForceCloseFee`.
+//
+// NOTE: This test does not check for a specific fee rate because channel force
+// closures should be bumped taking a budget into account not a specific
+// fee rate.
+func testBumpForceCloseFee(ht *lntest.HarnessTest) {
+	// Skip this test for neutrino, as it's not aware of mempool
+	// transactions.
+	if ht.IsNeutrinoBackend() {
+		ht.Skipf("skipping BumpForceCloseFee test for neutrino backend")
+	}
+	// fundAmt is the funding amount.
+	fundAmt := btcutil.Amount(1_000_000)
+
+	// We add a push amount because otherwise no anchor for the counter
+	// party will be created which influences the commitment fee
+	// calculation.
+	pushAmt := btcutil.Amount(50_000)
+
+	openChannelParams := lntest.OpenChannelParams{
+		Amt:     fundAmt,
+		PushAmt: pushAmt,
+	}
+
+	// Bumping the close fee rate is only possible for anchor channels.
+	cfg := []string{
+		"--protocol.anchors",
+	}
+
+	// Create a two hop network: Alice -> Bob.
+	chanPoints, nodes := createSimpleNetwork(ht, cfg, 2, openChannelParams)
+
+	// Unwrap the results.
+	chanPoint := chanPoints[0]
+	alice := nodes[0]
+
+	// We need to fund alice with 2 wallet inputs so that we can test to
```

Comment:
This does remind me something - that we need to check how many UTXOs are there and return an error if it's below 2, otherwise we can only bump one of the anchors below, and the error will only be shown in the logs, which can be a bad UX. Not in this PR tho as we will soon merge it.

Reply:
Ahh ok so you mean we would need 3 wallet inputs in the above case, so that we can be sure that we have enough wallet inputs to at least try to publish the anchor transaction.

Hmm however I think one input for all the anchor sweeps is enough, because as soon as one succeeds we can be sure the others won't make it into the mempool anyways ?

I added a `TODO` to investigate it.

Reply:
For full nodes yes, for neutrino nope - for instance when a local force close happens, and we only have one wallet utxo, that utxo could be used to bump the remote anchor, while an error won't be returned there.

Reply:
Makes sense, you mean we should generalize this behavior because this test for example skips the neutrino case, however you are right if we are not generalizing it we might run into this problem when all of a sudden we want to support the neutrino case here.